pkg/operator/controllers/monitoring/monitoring_controller_test.go

```diff
@@ -165,3 +216,109 @@ somethingElse:
 		})
 	}
 }
+
+func TestReconcilePVC(t *testing.T) {
+	log := logrus.NewEntry(logrus.StandardLogger())
+	tests := []struct {
+		name       string
+		reconciler func() *Reconciler
```

Comment:
Should we replace this (and refactor the older test for consistency) with `*fake.Clientset`? Because the only thing that chances in reconciler is fake client.
We can instantiate the reconciler itself in the test body.

Reply:
Refactored.  Should be below instead of reconcile function.  
```
kubernetescli kubernetes.Interface
```